impl/src/testing.rs

```diff
@@ -295,35 +293,14 @@ pub fn mock_crate_index(
 }
 
 /** Generate some basic metadata with an injected mock dependency */
-pub fn dummy_modified_metadata() -> Metadata {
-  let mut metadata = dummy_raze_metadata().metadata.clone();
+pub fn dummy_modified_metadata() -> RazeMetadata {
```

Comment:
For this change I needed to update the tests using it and opted to remove this function and add `dummy_modified_metadata.json.template`  which is a direct dump (with template's being added for things like system paths) of the metadata generated by this file before.

Reply:
I've decided to spin this off into https://github.com/google/cargo-raze/pull/321

When that's merged I'll be rebasing these changes on top of that @acmcarther just FYI